config/config.example.yml

```diff
@@ -23,10 +23,21 @@ ssr:
   # Determining which styles are critical is a relatively expensive operation; this option is
   # disabled (false) by default to boost server performance at the expense of loading smoothness.
   inlineCriticalCss: false
-  # Path prefixes to enable SSR for. By default these are limited to paths of primary DSpace objects.
-  # NOTE: The "/handle/" path ensures Handle redirects work via SSR.  The "/reload/" path ensures
-  # hard refreshes (e.g. after login) trigger SSR while fully reloading the page.
-  paths: [ '/home', '/items/', '/entities/', '/collections/', '/communities/', '/bitstream/', '/bitstreams/', '/handle/', '/reload/' ]
+  # Patterns to be run as regexes against the path of the page to check if SSR is allowed.
+  # If the path match any of the regexes it will be served directly in CSR.
+  # By default, excludes community and collection browse, global browse, global search, community list, and statistics.
```

Comment:
Now that we've added more patterns, we should update this sentence to end with "community list, statistics and various administrative tools"